index.html

```diff
@@ -540,45 +547,44 @@
               required ImageBitmap? outputImageBitmap;
           };
         </pre>
-        <div>
-          <p>
-            The <dfn>outputImageBitmap</dfn> attribute must return the value it
-            was initialized to. When the object is created, this attribute must
-            be initialized to null. It represents the <a>ImageBitmap</a> object
-            which on setting, must cause the user agent to run the following
-            steps:
-          </p>
-        </div>
-        <ol>
-          <li>Let <var>output bitmap</var> be the <a>ImageBitmap</a> object
-          assigned to the <a>outputImageBitmap</a> attribute.
-          </li>
-          <li>Let <var>output media stream track</var> be the
-          <a>MediaStreamTrack</a> returned by the
-          <code><a>addVideoProcessor</a>()</code> method.
-          </li>
-          <li>
-            <!-- TODO: this should be defined better. -->
-            Set <var>output bitmap</var> as the <a>source</a> of the
-            <var>output media stream track</var>.
-          </li>
-        </ol>
+      <div>
         <p>
-          When a user agent is required to <dfn>fire a video processor
-          event</dfn>, it must <a>fire a video worker event</a> named
-          <a>videoprocess</a>.
+          The <dfn>outputImageBitmap</dfn> attribute must return the value it
```

Comment:
`trackID`, `playbackTime` and `inputImageBitmap` use `<dfn><code>...</code></dfn>`. Should `outputImageBitmap` use the same style?